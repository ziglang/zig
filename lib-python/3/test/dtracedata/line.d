python$target:::line
/(copyinstr(arg1)=="test_line")/
{
    printf("%d\t%s:%s:%s:%d\n", timestamp,
        probename, basename(copyinstr(arg0)),
        copyinstr(arg1), arg2);
}
