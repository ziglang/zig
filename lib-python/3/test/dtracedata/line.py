def test_line():
    a = 1
    print('# Preamble', a)
    for i in range(2):
        a = i
        b = i+2
        c = i+3
        if c < 4:
            a = c
        d = a + b +c
        print('#', a, b, c, d)
    a = 1
    print('# Epilogue', a)


if __name__ == '__main__':
    test_line()
