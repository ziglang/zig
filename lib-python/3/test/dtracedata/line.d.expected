line:line.py:test_line:2
line:line.py:test_line:3
line:line.py:test_line:4
line:line.py:test_line:5
line:line.py:test_line:6
line:line.py:test_line:7
line:line.py:test_line:8
line:line.py:test_line:9
line:line.py:test_line:10
line:line.py:test_line:11
line:line.py:test_line:4
line:line.py:test_line:5
line:line.py:test_line:6
line:line.py:test_line:7
line:line.py:test_line:8
line:line.py:test_line:10
line:line.py:test_line:11
line:line.py:test_line:4
line:line.py:test_line:12
line:line.py:test_line:13
